### PR TITLE
[Silabs] Change freertos tick to 1024Hz

### PR DESCRIPTION
**Pull request by mkardous-silabs**
#### Description
Change freertos tick to be a multiple of the underlying clocks frequency

#### Tests
Manual tests to validate freertos's timer accuracy

### CODE REVIEW

**Review by cliffamzn — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

File: src/platform/silabs/CHIPDevicePlatformConfig.h [hunk 1]
```diff
@@ -91,11 +91,11 @@
 #endif // CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE
 
 #ifndef CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL
-#define CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL chip::System::Clock::Milliseconds32(SL_SLEEP_TIME_MS)
```
Comment (jepenven-silabs) [inline src/platform/silabs/CHIPDevicePlatformConfig.h:94]: This would have to be reverted. It breaks the feature of configuring the sleep time.

**Review by jepenven-silabs — CHANGES_REQUESTED** [review]

File: src/platform/silabs/CHIPDevicePlatformConfig.h (hunk 1 above)
Comment (mkardous-silabs) [reply src/platform/silabs/CHIPDevicePlatformConfig.h]: done

**Review by jepenven-silabs — APPROVED** [review]

File: src/platform/silabs/efr32/rs911x/BLEManagerImpl.cpp [hunk 2]
```diff
@@ -252,11 +252,11 @@ CHIP_ERROR BLEManagerImpl::_Init()
     mServiceMode = ConnectivityManager::kCHIPoBLEServiceMode_Enabled;
 
     // Create FreeRTOS sw timer for BLE timeouts and interval change.
-    sbleAdvTimeoutTimer = xTimerCreate("BleAdvTimer",             // Just a text name, not used by the RTOS kernel
-                                       BLE__DEFAULT_TIMER_PERIOD, // == default timer period (mS)
-                                       false,                     // no timer reload (==one-shot)
-                                       (void *) this,             // init timer id = ble obj context
-                                       BleAdvTimeoutHandler       // timer callback handler
+    sbleAdvTimeoutTimer = xTimerCreate("BleAdvTimer",                            // Just a text name, not used by the RTOS kernel
+                                       pdMS_TO_TICKS(BLE__DEFAULT_TIMER_PERIOD), // == default timer period (mS)
```
Comment (selissia) [inline src/platform/silabs/efr32/rs911x/BLEManagerImpl.cpp:256]: The comment is incorrect, this parameter is specified in ticks, not mS.

Comment (mkardous-silabs) [reply src/platform/silabs/efr32/rs911x/BLEManagerImpl.cpp]: done

**Pull request merged**